autoshop/core/templates/dashboard.html

```diff
@@ -2,7 +2,7 @@
 
 {% block content %}
 <h1>
-    SUCCESS
+    Dashboard
```

Comment:
Remove this file all together. Redirect the user to `index` page which will serve as `homepage`.